node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -180,7 +180,11 @@ pub struct GenesisConfig {
     pub(crate) name: String,
     pub(crate) timestamp: Timestamp,
     pub(crate) validator_slots: u32,
+    /// Number of eras before an auction actually defines the set of validators.
```

Comment:
Can you add the clarification here, too, to prevent the off-by-one error?
(My understanding is that in this case, if you bond with a sufficient bid in era `N`, you will be a validator in era `N + auction_delay + 1`, not in `N + auction_delay`?)

Reply:
@mpapierski Can you clarify/confirm?

Reply:
(It was my reading of [his earlier comment](https://github.com/CasperLabs/casper-node/pull/403#discussion_r513315345).)

Reply:
My mistake, I've updated accordingly